utils/transaction.py

```diff
@@ -55,3 +62,41 @@ def transformTransaction(txn):
 
 
 
+def blockTransform(block):
```

Comment:
Please move this to a new file `block.py` and leave this one reserved for functions relating explicitly to transactions.